py_func/src/lib.rs

```diff
@@ -0,0 +1,47 @@
+use gandiva_rust_udf_macro::udf;
+use pyo3::prelude::*;
+use pyo3::types::PyDict;
+
+#[udf]
+pub fn py(script: &str, arg: &str) -> String {
```

Comment:
Would this lead to code injection?

Reply:
Yes, this is possible. The `py` function is not registered by default, and we could move it into a experimental repo later.